project/terra_layer/models.py

```diff
@@ -164,6 +167,8 @@ class LayerGroup(models.Model):
     )
     order = models.IntegerField(default=0)
     exclusive = models.BooleanField(default=False)
+    byVariable = models.BooleanField(default=False)
```

Comment:
```suggestion
    by_variable = models.BooleanField(default=False)
```
respect the default python lower_case

Reply:
✅